client/data/agency-dashboard/use-fetch-dashboard-sites.ts

```diff
@@ -54,6 +54,7 @@ const useFetchDashboardSites = (
 					errorNotice( translate( 'Failed to retrieve your sites. Please try again later.' ) )
 				),
 			enabled: isPartnerOAuthTokenLoaded,
+			staleTime: 1000 * 60,
```

Comment:
We are updating the stale time for dashboard sites API to 1 min to be on the safer side since we are optimistically updating the notification settings.